pkgs/top-level/all-packages.nix

```
@@ -8635,7 +8635,11 @@ in
 
   flutterPackages =
     recurseIntoAttrs (callPackage ../development/compilers/flutter { });
-  flutter = flutterPackages.stable;
+  flutter = {
+    stable = flutterPackages.stable;
+    beta = flutterPackages.beta;
+    dev = flutterPackages.dev;
+  };
```

usually we don't define new attrsets in `all-packages.nix`, would you mind applying the following?
```suggestion
  flutter = flutterPackages.stable;
  flutterBeta = flutterPackages.beta;
  flutterDev = flutterPackages.dev;
```
besides that, LGTM

---

Thanks! Done @marsam